### PR TITLE
docs: fix example json usage in readme

### PR DESCRIPTION
**Pull request by purarue**
need to actually return the dict from the cleanup function

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -120,13 +120,15 @@ from bleanser.core.modules.json import JsonNormaliser, delkeys, Json
 
 
 class Normaliser(JsonNormaliser):
-    # here, j is a dict, each file that this gets passed from the CLI call below
-    # is pre-processed by the cleanup function
+    # here, j is a dict, each file that this gets passed from the CLI call
+    # below is pre-processed by the cleanup function
     def cleanup(self, j: Json) -> None:
```
Comment (karlicoss) [inline README.md]: I guess return type should be `Json` as well?

Comment (purarue) [reply README.md]: oh, yeah, it should

**Comment by karlicoss** [comment]
In principle, modifying in place could work too, but I figured in theory one might want to transform the json somehow and return a completely separate object (and it would be kinda awkward to overwrite the one passed as argument)

**Comment by purarue** [comment]
Yeah, could even check if the user returned None (or just didnt return) and in that case just use the old reference for `j` in the json normalizer, could prevent user error in which they just `del` some key manually and expected that to work

if they were using mypy it would warn them but they may not be

**Comment by karlicoss** [comment]
hmm yeah good idea!

**Pull request merged**